BLSQ/iaso
# POLIO-1068:  add mobile payment to the dropdown in budget tab

A new option mobile payment was missing in payment mode

Related JIRA tickets : POLIO-1068

## Self proofreading checklist


## Changes

- updated model to add a new payment mode
- added the option in dropdown
- added translation

## How to test

- Go to campaigns
- Create or edit a campaign
- Go to Budget Tab in Funds release column and choose the new option "Mobile Payment" 
- Save the campaign and check you receive the right value in api response

## Print screen / video

https://github.com/BLSQ/iaso/assets/25134301/e86b38af-bfae-434d-9ca5-36b8b04d95b9
